charts/fluent-bit/templates/_helpers.tpl

```diff
@@ -136,9 +136,3 @@ Create the name of OpenShift SecurityContextConstraints to use
 {{- printf "%s" (default (include "fluent-bit.fullname" .) .Values.openShift.securityContextConstraints.name) -}}
 {{- end -}}
 {{- end -}}
-{{/*
-Create the uid for grafana fluent-bit dashboard
-*/}}
-{{- define "fluent-bit.dashboard.uid" -}}
-{{- sha256sum (printf "%s/%s" .Release.Namespace .Release.Name) }}
```

Comment:
may be just add '| trunc 40' ?

{{- sha256sum (printf "%s/%s" .Release.Namespace .Release.Name) | trunc 40 }}

Reply:
yes, that was my first approach: https://github.com/fluent/helm-charts/pull/509/commits/ff6dcebe519f67bbe701dcfc0fbfdf668757ac68

as grafana does not need a prefilled uid anymore I think we should keep it as simple as possible - that's why I removed it completely after this discussion